pkg/logs/input/container/scanner_windows.go

```diff
@@ -0,0 +1,25 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+// +build windows
```

Comment:
you don't need the `+build` flag is you use `_windows` in the name

Reply:
I am not sure, check this out https://ci.appveyor.com/project/Datadog/datadog-agent/build/ajacquemot/base.1572